### DISCUSSION TITLE
how to add multiple tags system?

### DISCUSSION THREAD

**Discussion by H-Jett** [cotes2020/jekyll-theme-chirpy#2093] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### Is your feature request related to a problem? Please describe

I want to add multiple tags system, such as `location_tags`, `weather_tags` and so on. How to implement this feature ？

### Describe the solution you'd like

I want to add multiple tags system, such as `location_tags`, `weather_tags` and so on. How to implement this feature ？

**Comment by SoumyaK4** [comment]
Check the _layouts directory 
Look for files with tag or category in name...
And learn a bit about jekyll collections and you should be able to do it
